src/main/java/org/traccar/web/WebServer.java

```diff
@@ -100,6 +101,16 @@ protected void handleErrorPage(
         handlers.addHandler(servletHandler);
         handlers.addHandler(new GzipHandler());
         server.setHandler(handlers);
+
+        if (config.getBoolean("web.logRequests")) {
+          NCSARequestLog requestLog = new NCSARequestLog("./logs/jetty-yyyy_mm_dd.request.log");
```

Comment:
Log file location has to be configurable.

Reply:
ok